[2025-04-01T14:27:04.176+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-01T14:27:04.327+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_dag_name.create_table scheduled__2021-02-06T00:00:00+00:00 [queued]>
[2025-04-01T14:27:04.378+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_dag_name.create_table scheduled__2021-02-06T00:00:00+00:00 [queued]>
[2025-04-01T14:27:04.379+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-01T14:27:04.501+0000] {taskinstance.py:2890} INFO - Executing <Task(BigQueryCreateTableOperator): create_table> on 2021-02-06 00:00:00+00:00
[2025-04-01T14:27:04.633+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_dag_name', 'create_table', 'scheduled__2021-02-06T00:00:00+00:00', '--job-id', '89', '--raw', '--subdir', 'DAGS_FOLDER/con.py', '--cfg-path', '/tmp/tmp2vjo6gf4']
[2025-04-01T14:27:04.639+0000] {standard_task_runner.py:105} INFO - Job 89: Subtask create_table
[2025-04-01T14:27:04.623+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=449) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-01T14:27:04.645+0000] {standard_task_runner.py:72} INFO - Started process 503 to run task
[2025-04-01T14:27:04.972+0000] {task_command.py:467} INFO - Running <TaskInstance: my_dag_name.create_table scheduled__2021-02-06T00:00:00+00:00 [running]> on host 900d630d7d5f
[2025-04-01T14:27:06.213+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_dag_name' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2021-02-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-02-06T00:00:00+00:00'
[2025-04-01T14:27:06.231+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-04-01T14:27:06.231+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-04-01T14:27:06.232+0000] {logging_mixin.py:190} INFO - Current task name:create_table state:running start_date:2025-04-01 14:27:04.330299+00:00
[2025-04-01T14:27:06.232+0000] {logging_mixin.py:190} INFO - Dag name:my_dag_name and current dag run status:running
[2025-04-01T14:27:06.251+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-01T14:27:06.452+0000] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-04-01T14:27:06.594+0000] {base.py:84} INFO - Retrieving connection 'gcp_co'
[2025-04-01T14:27:06.631+0000] {bigquery.py:1274} INFO - Creating table...
[2025-04-01T14:27:07.753+0000] {bigquery.py:1304} INFO - Table fake_news_detection.training_data already exists.
[2025-04-01T14:27:07.947+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-01T14:27:07.963+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_dag_name, task_id=create_table, run_id=scheduled__2021-02-06T00:00:00+00:00, execution_date=20210206T000000, start_date=20250401T142704, end_date=20250401T142707
[2025-04-01T14:27:08.131+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-04-01T14:27:08.145+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-04-01T14:27:08.146+0000] {logging_mixin.py:190} INFO - Dag name:my_dag_name queued_at:2025-04-01 14:26:02.500427+00:00
[2025-04-01T14:27:08.151+0000] {logging_mixin.py:190} INFO - Task hostname:900d630d7d5f operator:BigQueryCreateTableOperator
[2025-04-01T14:27:08.271+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-01T14:27:08.425+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-01T14:27:08.437+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
